.github/workflows/release.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
           go-version: '${{ env.GO_VERSION }}'
 
       - name: build release for all architectures
-        run: SKIP_VERSION_CHECK=1 make release tag=${{ env.RELEASE_VERSION }}
+        run: make release tag=${{ env.RELEASE_VERSION }}
```

Comment:
To make this work, we need to update the version check in `scripts/release.go`. We should replace the build and invocation with `--version` with just calling the tag given as a parameter matches the one output by `get_git_tag_name`.